src/test/java/io/tackle/pathfinder/controllers/AssessmentsResourceTest.java

```diff
@@ -651,7 +651,7 @@ public void given_ApplicationAssessed_When_CopyAssessmentToAnotherNonAssessedApp
 				.ignoringCollectionOrder()
 				.isEqualTo(assessmentSource);
 	}
-
+	@Test
```

Comment:
Could you please add an empty line between `}` and `@Test`?

Reply:
:+1: